src/servers/ZoneServer2016/entities/vehicle.ts

```diff
@@ -163,33 +164,61 @@ function getHotwireEffect(vehicleId: VehicleIds) {
 }
 
 export class Vehicle2016 extends BaseLootableEntity {
+  /** Used for determining seat swap logic*/
```

Comment:
Not really if i’m not mistaken it’s to know if the vehicle physics are currently managed by a client